### PR TITLE
add getSessionWithSpace

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: connection_pool.go [hunk 1]
```diff
@@ -118,6 +118,23 @@ func (pool *ConnectionPool) GetSession(username, password string) (*Session, err
 	return &newSession, nil
 }
 
+func (pool *ConnectionPool) GetSessionWithSpace(username, password, space string) (*Session, error) {
+	session, err := pool.GetSession(username, password)
+	if err != nil {
+		return nil, err
+	}
+	stmt := "USE" + space
+	res, err := session.Execute(stmt)
```
Comment (Aiee) [inline connection_pool.go:127]: Add a space between "USE" and space name

Comment (Aiee) [reply connection_pool.go:127]: Use `tryToExecute(session, stmt)`

**Comment by Aiee** [comment]
We could attach the space with a connection pool instead of a session.

**Comment by everywan** [comment]
Hey guys, i need this too, and prepare write the session pool. Maybe we are doing the duplicate work.

Detail in [here](https://github.com/zhihu/norm/blob/master/docs/dialector.adoc#nebula-connection-%E4%BB%8B%E7%BB%8D)

**Pull request closed without merging**